_src/_assets/js/dashboard-charts.js

```diff
@@ -1,11 +1,15 @@
 /* eslint-disable */
 
-(function loadAllCharts() {
+;(function loadAllCharts() {
   const formatDate = d3.timeFormat('%b. %e')
   const parseDate = d3.timeParse('%Y%m%d')
   const usDailyReq = d3.json('https://covidtracking.com/api/us/daily')
   const stateDailyReq = d3.json('https://covidtracking.com/api/states/daily')
 
+  const cdcDailyReq = d3.json(
+    'https://spreadsheets.google.com/feeds/list/16gBHQ7dCJK1psqEMasmLKiFlzoNKcfNujVpmHLHldSY/od6/public/values?alt=json',
```

Comment:
Is this gonna end up in the API? I'd hate for our whole page to go down just because of Google sheets but if its where the data lives, its where the data lives.